scripts/install.sh

```diff
@@ -216,7 +216,30 @@ function outro() {
     fi
     printf "\n"
     printf "\n"
-    printf "Node join commands expire after two hours. You can rerun the kURL install script on any master node to generate fresh node join commands.\n"
+    
+    local prefix="curl -sSL${proxyFlag} $KURL_URL/$INSTALLER_ID/"
+    if [ -z "$KURL_URL" ]; then
+        prefix="cat "
+    fi
+
+    if [ "$HA_CLUSTER" = "1" ]; then
+        printf "Master node join commands expire after two hours, and worker node join commands expire after 24 hours.\n"
+        printf "\n"
+        if [ "$AIRGAP" = "1" ]; then
+            printf "To generate new node join commands, run ${GREEN}cat ./tasks.sh | sudo bash -s join_token ha airgap${NC} on an existing master node.\n"
```

Comment:
Curious, but why the inconsistent use of ${prefix}?

Reply:
I copied it from the airgap vs online install scripts below this